tests/migrations/test_commands.py

```diff
@@ -1434,6 +1435,22 @@ class Meta:
             self.assertIn("You can accept the default 'timezone.now' by pressing 'Enter'", prompt_output)
             self.assertIn("Add field creation_date to entry", output)
 
+    @unittest.skipUnless(connection.vendor == 'postgresql', "Test only for PostgreSQL")
+    def test_makemigrations_for_class_serializable_model(self):
+        """
+        Python internal built-in modules rename in particular
+        """
+        self.assertTableNotExists('migrations_unicodemodel')
+        apps.register_model('migrations', ClassSerializableModel)
+        with self.temporary_migration_module() as migration_dir:
+            call_command("makemigrations", "migrations", verbosity=0)
+            # Check for existing 0001_initial.py file in migration folder
+            initial_file = os.path.join(migration_dir, "0001_initial.py")
+            self.assertTrue(os.path.exists(initial_file))
+            with open(initial_file, 'r', encoding='utf-8') as fp:
+                content = fp.read()
+                self.assertIn('psycopg2.extras.DateTimeTZRange', content)
```

Comment:
This migration integration test test is unnecessary. We usually only add unit tests to the `test_writer` module when adding support for a new kind of serialization.